website-custom-views/src/content/api-reference/custom-view-config.mdx

```diff
@@ -176,17 +176,15 @@ For more information, see [Granular permissions for local development](/help-nee
 
 ## `env.development.hostUriPath`
 
-The Custom View [context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) will contain the ["hostUrl"](https://docs-beta-custom-views.commercetools.vercel.app/custom-views/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property which holds the value of the host built-in application view where the user opens it.
+The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/<custom-view-host>` by default.
```

Comment:
The value the actually `@@custom-view-host@@`

```suggestion
The [Custom View context](/api-reference/commercetools-frontend-application-shell-connectors#custom-view-context) contains the [`hostUrl`](/api-reference/commercetools-frontend-application-shell-connectors#hosturl) property with the value of the built-in application's current URL. When working locally it is set to `http://localhost:3001/<projectKey>/@@custom-view-host@@` by default.
```

Reply:
Thanks @emmenko. Can you share why we use this format specifically @@?

Reply:
It's the actual value, it's not a placeholder.

This is essentially the equivalent of Custom Applications `entryPointUriPath`. However, Custom Views don't need an entry point but to keep using the same logic and infrastructure as for Custom Applications we decided to define a special entry point value for Custom Views.
That's why it has this special format.

Reply:
Got it. Thanks, Nicola for the explanation.